package.json

```diff
@@ -8,8 +8,14 @@
 	"module": "dist/tween.esm.js",
 	"exports": {
 		".": {
-			"import": "./dist/tween.esm.js",
```

Comment:
could use more easy way by just one line inserted like this ?
![image](https://github.com/tweenjs/tween.js/assets/920487/ed49aeea-e2a0-4be0-8649-248818d09135)
it seems work for me!